src/actions/auth.js

```diff
@@ -3,7 +3,7 @@
 import ax from 'axios';
 import { GET_USERDATA, PROFILE_CHANGE, GROUP_CHANGE, GET_NEWS } from './types';
 
-const axios = ax.create({
+export const axios = ax.create({
```

Comment:
I think it should be another module. For exaample http.js or http_client.js ...

Reply:
You can use **export default** after the assignment.
const session = ax.create(...
export default session

Reply:
done in e4f070d